packages/integration-sdk-core/src/data/createIntegrationRelationship.ts

```diff
@@ -142,38 +140,69 @@ export function createIntegrationRelationship(
     | MappedRelationshipOptions
     | MappedRelationshipLiteralOptions,
 ): Relationship {
-  if ('_mapping' in options) {
+  if ('_mapping' in options || 'target' in options) {
     return createMappedRelationship(options);
-  } else if ('target' in options) {
-    return createMappedRelationship({
+  } else {
+    return createDirectRelationship(options);
+  }
+}
+
+/**
+ * Create a direct `IntegrationRelationship` between two entities
+ *
+ * `DirectRelationshipOptions` is recommended over `DirectRelationshipOptionsLiteral`. Older integrations may need to use the literal forms to control values for some reason or other.
+ */
+export function createDirectRelationship(
+  options: DirectRelationshipOptions | DirectRelationshipLiteralOptions,
+): ExplicitRelationship {
+  //
+  if ('fromType' in options) {
+    return createRelationship(options);
+  //
+  } else {
+    return createRelationship({
+      _class: options._class,
+      fromType: options.from._type,
+      fromKey: options.from._key,
+      toType: options.to._type,
+      toKey: options.to._key,
+      properties: options.properties,
+    });
+  }
+}
+
+/**
+ * Create a mapped `IntegrationRelationship`.
+ *
+ * `MappedRelationshipOptions` is recommended over `MappedRelationshipOptionsLiteral`. Older integrations may need to use the literal forms to control values for some reason or other.
+ */
+export function createMappedRelationship(
+  options: MappedRelationshipOptions | MappedRelationshipLiteralOptions,
+): MappedRelationship {
+  // MappedRelationshipLiteralOptions
+  if ('_mapping' in options) {
+    return createMappedRelationshipLiteral(options);
+  // MappedRelationshipOptions
```

Comment:
```suggestion
  if ('_mapping' in options) {
    return createMappedRelationshipLiteral(options);
```
To match the other function where removing the empty comments is suggested 😁  I don't feel these add to the clarity of the function.